server/src/actors/project.rs

```diff
@@ -405,12 +410,12 @@ impl GetEventAction {
 }
 
 /// Indicates what should happen to events based on their meta data.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub enum EventAction {
     /// The event should be discarded.
-    Discard,
+    Discard(OutcomeReason),
```

Comment:
Please check if this can have just an `OutcomeInvalidReason`. We can consider to rename it to something like `DiscardReason`. This would lead to a variant `Outcome::Invalid(DiscardReason)`, which might be acceptable.

Reply:
Ok, done